assets/css/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+    background-image: url("../images/connectwork.png");
+    background-repeat: repeat;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+h1 {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    font-size: 40px;
+    border-bottom: 2px solid black;
+    padding-top: 30px;
+    padding-bottom: 10px;
+    margin-bottom: 10px;
+}
+
+p {
+    padding-bottom: 100px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+    width: 960px;
+    height: 700px;
+    margin-bottom: 120px;
+    margin-left: auto;
+    margin-right: auto;
+    background-image: url("../images/ep_naturalwhite.png");
+    background-repeat: repeat;
+}
+
+.content {
+    margin-left: 50px;
+    margin-right: 50px;
+}
+
+.git,
+.linkedin,
+.email {
+    width: 100px;
+    height: 100px;
+    margin-left: 50px;
```

Comment:
prefer condensed version of margin when you can use it:
`margin: 15px 0px 92px 50px;`